latch_cli/services/register/models.py

```diff
@@ -187,16 +205,19 @@ def image_tagged(self):
         # TODO (kenny): check version is valid for docker
```

Comment:
wanna add this to this pr, should be a simple regex

Reply:
from the docker docs:

> A tag name must be valid ASCII and may contain lowercase and uppercase letters, digits, underscores, periods and dashes. A tag name may not start with a period or a dash and may contain a maximum of 128 characters.

Reply:
k